app/components/FFXIVResults/ShortagePredictor/Results.tsx

```
@@ -0,0 +1,276 @@
+              {Object.entries(WorldList).map(([dataCenter, worlds]) => (
+                <div key={dataCenter}>
+                  <p className="text mt-1 font-semibold text-gray-800">
+                    {dataCenter}
+                  </p>
+                  {worlds.map(({ name }) => {
+                    const isSelected = state.exportServers.includes(name)
+
+                    return (
+                      <CheckBox
+                        key={dataCenter + name + state.exportServers}
+                        labelTitle={'-- ' + name}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)